src/main/resources/application.yaml

```diff
@@ -98,6 +100,8 @@ ftp:
       folder: PROBATE
     - service: sscs
       folder: SSCS
+    - service: finrem_document_generator
+      folder: FINREM
```

Comment:
FINREM folder is currently not configured by our scanning supplier so will keep this PR on hold for now

Reply:
ok

Reply:
to be fair, theoretically, it can go in. just don't post anything otherwise letters will become stale which will cause alerts. to avoid that - best to configure ftp folder first as @nitinprabhu pointed out